lms/djangoapps/organizations/views.py

```diff
@@ -96,8 +106,13 @@ def users(self, request, pk):
             include_course_counts = request.QUERY_PARAMS.get('include_course_counts', None)
             include_grades = request.QUERY_PARAMS.get('include_grades', None)
             course_id = request.QUERY_PARAMS.get('course_id', None)
+            view = request.QUERY_PARAMS.get('view', None)
             grade_complete_match_range = getattr(settings, 'GRADEBOOK_GRADE_COMPLETE_PROFORMA_MATCH_RANGE', 0.01)
             course_key = None
+            if view == 'ids':
```

Comment:
I moved `ids` formatter for users below the `include_course_counts` check moving further down increases number of unnecessary queries.